definition_schema.json

```diff
@@ -48,30 +48,104 @@
     "_abbrev": {
       "type": "string"
     },
-    "_examples": {
-      "type": "array"
-    },
-    "_history": {
-      "type": "array"
-    },
-    "_links": {
-      "type": "object"
-    },
     "_name": {
       "type": "string"
     },
     "_version": {
       "type": "string"
     },
-    "properties": {
-      "type": "object"
-    },
     "additionalProperties": {
       "type": "boolean"
     },
     "_format": {
       "type": "string"
     },
+    "_examples": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "title": {
+            "type": "string"
+          },
+          "url": {
+            "type": "string",
+            "format": "uri-reference"
+          }
+        },
+        "required": [
+          "title",
+          "url"
+        ],
+        "additionalProperties": false
+      }
+    },
+    "_history": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "version": {
+            "type": "string"
+          },
+          "introduced_in": {
+            "type": "string"
+          },
+          "changes": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "version"
+        ]
+      }
+    },
+    "_links": {
+      "type": "object",
+      "patternProperties": {
+        "^[A-Z_]+$": {
+          "type": "object",
+          "properties": {
```

Comment:
`experimental` is missing from this list of properties.

Reply:
Good catch. I added in some `"additionalProperties": false` to catch errors like that and also added in `experimental`. See [15ac806](https://github.com/eiffel-community/eiffel/pull/402/commits/15ac806fc609fe728f0291087b2a13e9c17d3602)